model/buildlogger.go

```diff
@@ -226,6 +227,19 @@ func (l *Log) appendLogChunkInfo(ctx context.Context, logChunk LogChunkInfo) err
 		"logChunkInfo": logChunk,
 		"op":           "append log chunk info to buildlogger log",
 	})
+
+	if err = l.env.GetStatsCache(cedar.StatsCacheBuildlogger).AddStat(cedar.Stat{
+		Count:   logChunk.NumLines,
+		Project: l.Info.Project,
+		Version: l.Info.Version,
+		Task:    l.Info.TaskID,
+	}); err != nil {
+		grip.ErrorWhen(sometimes.Percent(10), message.WrapError(err, message.Fields{
```

Comment:
Since we do not expect these errors often, I think it's better to always log when an error occurs. It may help us diagnose issues or, if this is erroring frequently, realize that our stats probably aren't very useful since so many are missing.